test/correctness/truncated_pyramid.cpp

```diff
@@ -20,47 +20,39 @@ int main(int argc, char **argv) {
 
     Func pyr_down[levels];
     Param<int> width, height;
-    vector<pair<Expr, Expr> > sizes(levels);
-    sizes[0] = { width, height };
+    vector<pair<Expr, Expr>> sizes(levels);
+    sizes[0] = {width, height};
     for (int i = 1; i < levels; i++) {
-        sizes[i] = { (sizes[i-1].first + 1)/2, (sizes[i-1].second + 1)/2 };
+        sizes[i] = {(sizes[i - 1].first + 1) / 2, (sizes[i - 1].second + 1) / 2};
     }
     pyr_down[0] = input;
     for (int i = 1; i < levels; i++) {
-        Func bounded = BoundaryConditions::repeat_edge(pyr_down[i-1], {{0, sizes[i].first}, {0, sizes[i].second}});
+        Func bounded = BoundaryConditions::repeat_edge(pyr_down[i - 1], {{0, sizes[i].first}, {0, sizes[i].second}});
         // Some simple stencil that acts like a 4x4 kernel for the purpose of bounds inference.
         Func downsampled;
-        downsampled(x, y) = bounded(2*x-1, 2*y-1) + bounded(2*x+2, 2*y+2);
+        downsampled(x, y) = bounded(2 * x - 1, 2 * y - 1) + bounded(2 * x + 2, 2 * y + 2);
 
         // Only compute it if the pyramid level is large enough.
         pyr_down[i](x, y) = select(max(sizes[i].first, sizes[i].second) > 5,
                                    downsampled(x, y),
                                    0.0f);
 
         // Specialize it, to introduce another type of condition in the params.
-        pyr_down[i].compute_root()
-            .specialize(max(width, height) > 32)
-            .vectorize(x, 16)
-            .parallel(y, 16, TailStrategy::GuardWithIf);
+        pyr_down[i].compute_root().specialize(max(width, height) > 32).vectorize(x, 16).parallel(y, 16, TailStrategy::GuardWithIf);
```

Comment:
one scheduling directive per line. Perhaps this and the other case failed because of the operator[]